package.json

```diff
@@ -732,6 +732,22 @@
       ],
       "integrity": "AB4D44465EA5135912737F8D943DB8DB7ADF1CC213F433188401FEE36B78A319"
     },
+    {
+      "id": "Razor",
+      "description": "Razor Language Server (Platform Agnostic)",
+      "url": "https://download.visualstudio.microsoft.com/download/pr/f8b5b74b-3df3-47cc-83b1-cd1d93d1771d/e456a753103b84b87bf5f000499ce3d7/razorlanguageserver-platformagnostic-7.0.0-preview.23513.5.zip",
```

Comment:
This version doesn't exactly match the version above since `23516.2` wasn't published with a platform agnostic language server. This is temporary and should be resolved whenever Razor next updates its versions.

Reply:
as soon as https://github.com/dotnet/roslyn/pull/70568 merges we can bump razor and rosly, in this PR perhaps?